ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -882,6 +890,33 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
         }
     }
 
+    private lazy var unreadDotLayer: CALayer = {
+        let unreadDotLayer = CALayer()
+        let unreadDotSize = TableViewCellTokenSet.leadingDotDimensions
```

Comment:
Why are you referencing TableViewTokenSet directly here, rather than tokenSet?

Reply:
Any time I use TableViewTokenSet, that means I'm using a token constant! Which is what leadingDotDimensions happens to be.